virtcontainers/acrn.go

```diff
@@ -14,32 +14,58 @@ import (
 	"strings"
 	"syscall"
 	"time"
+	"unsafe"
 
 	opentracing "github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
 	"github.com/kata-containers/runtime/virtcontainers/device/config"
 	persistapi "github.com/kata-containers/runtime/virtcontainers/persist/api"
+	"github.com/kata-containers/runtime/virtcontainers/pkg/uuid"
 	"github.com/kata-containers/runtime/virtcontainers/store"
 	"github.com/kata-containers/runtime/virtcontainers/types"
 	"github.com/kata-containers/runtime/virtcontainers/utils"
 )
 
-// AcrnState keeps Acrn's state
-type AcrnState struct {
-	UUID string
-}
-
-// AcrnInfo keeps PID of the hypervisor
+// acrnUUIDsToIdx lists Idx corresponding to the UUID
+var acrnUUIDsToIdx = map[string]uint8{
+	"a7ada506-1ab0-4b6b-a0da-e513ca9b8c2f": 0,
```

Comment:
woow, this looks like black magic :smile: , could you please include a comment explaining what mean or from where did you get these IDs ?

Reply:
Added comments.